src/core/db/models.py

```diff
@@ -130,7 +130,6 @@ class UserTask(Base):
     class Status(str, enum.Enum):
         """Статус задачи у пользователя."""

```

Comment:
Нужно не просто удалить этот статус из модели, но и удалить его в методах, но сохранив их работоспособность. 
Например, 
```suggestion
    async def get_all_ids(
...
                    or_(UserTask.status == UserTask.Status.NEW, UserTask.status == UserTask.Status.UNDER_REVIEW),
```
или 
 ```suggestion
    async def get_members_ids_for_excluding(self, shift_id: UUID, task_amount: int) -> list[UUID]:
                    UserTask.status != UserTask.Status.NEW,
```

Reply:
И миграции сделать.

Reply:
Я правильно понимаю что предложенный метод должен возвращать description задания текущего календарного дня?

Reply:
методы изменил, конфликт исправил. С миграциями нужна помощь

Reply:
В readme.md есть подробное описание про миграции. Если что-то не будет получаться, то напишите в чём именно нужна помощь.

Reply:
@ADesBiysk миграцию сделал, можете оценить насколько корректно? Мб есть более простой вариант действий